app/javascript/packs/base.jsx

```diff
@@ -156,18 +156,12 @@ initializeNav();
 
 async function loadCreatorSettings() {
   try {
-    const [
```

Comment:
Since all functionality has been moved to the pack file, we no longer need this stimulus controller